examples/mnist/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:14.04.5
```

Comment:
This Dockerfile is not meant to be committed (unless we want to for some reason).

I'm just trying to track what I've done.